package.json

```diff
@@ -89,7 +87,6 @@
     "less-loader": "^2.2.1",
     "marked": "^0.3.5",
     "mocha": "^2.3.3",
-    "mt-changelog": "^0.6.2",
```

Comment:
Per https://github.com/AlexKVal/release-script/blob/05ab23673b040549bdece1802a262f3744c4beb6/src/release.js#L27-L32, this is enough to disable automated changelog generation.